<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectFileWindow</class>
 <widget class="QMainWindow" name="SelectFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select file</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="filepathLE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>561</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="editFileBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createNewFileBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Create new</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browseFilesBtn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
